experiment/oss_fuzz_checkout.py

```diff
@@ -483,10 +483,28 @@ def _build_image(project_name: str) -> str:
 def prepare_project_image(project: str) -> str:
   """Prepares original image of the |project|'s fuzz target build container."""
   image_name = f'gcr.io/oss-fuzz/{project}'
-  if (ENABLE_CACHING and is_image_cached(project, 'address') and
-      (_image_exists_locally(image_name, project_name=project) or
-       _image_exists_online(image_name, project_name=project))):
-    logger.info('Using cached project image for %s', project)
-    return image_name
+  if ENABLE_CACHING and is_image_cached(project, 'address'):
```

Comment:
should we move `prepare_project_image` into `container_tool`? container_tool seems to be the only user, and it's very specific to that.

Reply:
Logically I consider projects images as a feature provided by OSS-Fuzz, `contain_tool` uses this feature and focuses on executing commands / parsing outputs instead of handling images.
We can refine this and find the best solution when working on the GSoC project : )